### PR TITLE
SOC-3728 chagne first post author in Discussions for DE communities from ForestFar to Mira Laime

### PR DESCRIPTION
**Pull request by slayful**

### CODE REVIEW

File: extensions/wikia/Discussions/maintenance/StaffWelcomePoster.class.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
 class StaffWelcomePoster {
 
 	const LANG_TO_STAFF_MAP = [
-		'de' => 24961680,      // ForestFairy
+		'de' => 26339491,      // Mira Laime
```
Comment (WikiaTech) [inline extensions/wikia/Discussions/maintenance/StaffWelcomePoster.class.php:6]: INFO Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

File: extensions/wikia/Discussions/maintenance/StaffWelcomePoster.class.php [hunk 2]
```diff
@@ -3,7 +3,7 @@
...
 		'en' => 26339491,      // Mira Laime
```
Comment (WikiaTech) [inline extensions/wikia/Discussions/maintenance/StaffWelcomePoster.class.php:7]: INFO Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 6 issues
* MINOR 3 minor
* INFO 3 info

Watch the comments in this conversation to review them.

#### 4 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MINOR [StaffWelcomePoster.class.php#L18](https://github.com/Wikia/app/blob/2d6b89a2e07f5ce1ae447c00ea7e9470cff03868/extensions/wikia/Discussions/maintenance/StaffWelcomePoster.class.php#L18): Define a constant instead of duplicating this literal "zh-hans" 3 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [StaffWelcomePoster.class.php#L19](https://github.com/Wikia/app/blob/2d6b89a2e07f5ce1ae447c00ea7e9470cff03868/extensions/wikia/Discussions/maintenance/StaffWelcomePoster.class.php#L19): Define a constant instead of duplicating this literal "zh-hant" 3 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [StaffWelcomePoster.class.php#L41](https://github.com/Wikia/app/blob/2d6b89a2e07f5ce1ae447c00ea7e9470cff03868/extensions/wikia/Discussions/maintenance/StaffWelcomePoster.class.php#L41): Immediately return this expression instead of assigning it to the temporary variable "$success". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)
1. INFO [StaffWelcomePoster.class.php#L15](https://github.com/Wikia/app/blob/2d6b89a2e07f5ce1ae447c00ea7e9470cff03868/extensions/wikia/Discussions/maintenance/StaffWelcomePoster.class.php#L15): Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

**Comment by garthwebb** [comment]
Doubled checked identity of that user ID & its good.  LGTM.

**Comment by slayful** [comment]
Code moved to config.
https://github.com/Wikia/config/compare/IRIS-3728?expand=1

**Pull request closed without merging**